src/sentry/lang/native/symbolicator.py

```diff
@@ -344,6 +365,9 @@ def merge_symbolicator_image(raw_image, complete_image, sdk_info, handle_symboli
         elif not (v is None or (k, v) == ('arch', 'unknown')):
             raw_image[k] = v
 
+    if raw_image.get('debug_id'):
+        raw_image['debug_id'] = id_from_breakpad(raw_image['debug_id'])
```

Comment:
Where did this pop up? This should be handled by normalization.

Reply:
`id_from_breakpad` is also called in the symbolic Python bindings. We can also move this into symbolicator.

Normalization doesn't run after enhancers.